server/protocols/jmap/src/test/java/org/apache/james/jmap/model/UpdateMessagePatchTest.java

```diff
@@ -49,38 +46,6 @@ public void builderShouldSetUnreadFalseWhenBuiltWithIsUnreadFalse() {
         UpdateMessagePatch.builder().isUnread(false).build();
     }
 
-    @Test
-    public void applyStateShouldSetFlaggedOnlyWhenIsFlagged() {
```

Comment:
maybe you should move these tests instead of removing them as you move the function?